tools/featurecounts/README.rst

```diff
@@ -0,0 +1,55 @@
+FeatureCounts wrapper for Galaxy
```

Comment:
Deprecated
=========

IUC has a maintained build of featureCounts and SubRead + url